portable_concurrency/future

```diff
@@ -8,7 +8,7 @@
  * `future` API imlementation with continuations and executors support
  */
 
-#include "bits/alias_namespace.h"
+#include "functional"
```

Comment:
It's hard to track dependencies added through this header. It would be better to `#include "unique_function.hpp"` in the file which constructs `unique_function<void()>` instances from different function types. Right now it looks like `future_state.h` is proper file which should depend on `unique_function` implementation details.